emcc.py

```diff
@@ -2672,6 +2675,12 @@ def do_binaryen(target, asm_target, options, memfile, wasm_binary_target,
     cmd += shared.Building.get_binaryen_feature_flags()
     if debug_info:
       cmd += ['-g'] # preserve the debug info
+    if use_source_map(options):
+      cmd += ['--input-source-map=' + wasm_source_map_target]
+      cmd += ['--output-source-map=' + wasm_source_map_target]
```

Comment:
We need output-source-map here but not above?    I guess we should factor this out into a "get_binaryen_args()" function at some point?

Reply:
Where do you mean "above"?

Reply:
I mean when we call finalize..around line 2656

Reply:
the call on 2656 is for asm2wasm, so it's not for the wasm backend. there, there isn't a source map that arrived from the wasm backend that we need to keep passing through.

Reply:
OK makes sense.